test/integration/local/routing/update-requests-awaiting-manual-ledger-check.test.js

```diff
@@ -0,0 +1,146 @@
+const { resetDatabase, closeDatabaseConnection, savePaymentRequest } = require('../../../helpers')
+
+jest.mock('../../../../app/routing/get-schedule-id')
+const { getScheduleId: mockGetScheduleId } = require('../../../../app/routing/get-schedule-id')
+
+jest.mock('../../../../app/routing/transform-payment-request')
+const { transformPaymentRequest: mockTransformPaymentRequest } = require('../../../../app/routing/transform-payment-request')
+
+jest.mock('../../../../app/processing/account-codes')
+const { mapAccountCodes: mockMapAccountCodes } = require('../../../../app/processing/account-codes')
+
+jest.mock('../../../../app/processing/complete-payment-requests')
+const { completePaymentRequests: mockCompletePaymentRequests } = require('../../../../app/processing/complete-payment-requests')
+
+jest.mock('../../../../app/holds')
+const { removeHoldByFrn: mockRemoveHoldByFrn } = require('../../../../app/holds')
+
+jest.mock('../../../../app/event')
+const { sendProcessingRouteEvent: mockSendProcessingRouteEvent } = require('../../../../app/event')
+
+const schedule = require('../../../mocks/schedules/in-progress')
+
+const { AWAITING_LEDGER_CHECK } = require('../../../../app/constants/hold-categories-names')
+
+const { updateRequestsAwaitingManualLedgerCheck } = require('../../../../app/routing/update-requests-awaiting-manual-ledger-check')
+
+let paymentRequest
+let manualLedgerCheckResult
+
+describe('update requests awaiting manual ledger check', () => {
+  beforeEach(async () => {
+    jest.clearAllMocks()
+    paymentRequest = JSON.parse(JSON.stringify(require('../../../mocks/payment-requests/payment-request')))
+    schedule.scheduleId = 1
+    manualLedgerCheckResult = { paymentRequest, paymentRequests: [paymentRequest, paymentRequest] }
+    await resetDatabase()
```

Comment:
Does this await need to be defined here? Or can it be defined below the jest.clearAllMocks?

Reply:
Moved